### DISCUSSION TITLE
Azure subscription

### DISCUSSION THREAD

**Discussion by ajax159** [msusdev/Innovation-Challenge#3] [Q&A]
Will we have some type of Azure subscription provided by Microsoft or will everything be on us?

**Comment by MatthewCalder-msft** [comment]
I'm sorry but we are not able to provide Azure subscriptions for the hackathon. But you should be able to build a working proof of concept with the services that are free with the Azure free trial- the ones that are free for 12 months or always free. [You can find details if you scroll down on this page ](https://azure.microsoft.com/en-us/free/free-account-faq/ )

**Reply by CarnegieJ** [reply]
Another Azure low cost or free subscription option ...
For current enrolled students, there is also the option with [Azure for Students](https://azure.microsoft.com/free/students) (https://azure.microsoft.com/free/students).  Use your university or school email to sign up and renew each year you are a student. It will require an email address from your academic institution.
